### ISSUE TITLE
OpenCL partition size computation for indirect loops

### ISSUE THREAD

**Issue by gsigms** [OP2/PyOP2#32]
in `pyop2/opencl.py:ParLoopCall::_i_compute_partition_size()`

this method is supposed to compute the partition size for indirect loop calls based on the available local memory of the device. At the moment the computation fails to account for some memory requirements leading to too large partitions that exceed the available shared memory available. This is temporarily fixed by arbitrarily removing 512bytes of "available" local memory.

**Comment by wence-** [comment]
No longer relevant after #507.
